code/game/objects/items/food/frozen.dm

```diff
@@ -434,3 +434,16 @@
 	foodtypes = DAIRY | SUGAR
 	venue_value = FOOD_PRICE_NORMAL
 	crafting_complexity = FOOD_COMPLEXITY_3
+
+/obj/item/food/popsicle/meatsicle
+	name = "Meatsicle"
+	desc = "A horrific abomination of raw meat, glazed with sugar on a stick, then frozen."
```

Comment:
this is amazing

Reply:
https://media.tenor.com/CGDqgg8bb9AAAAAC/meat-popsicle-5th-element.gif